### PR TITLE
Allow voting on non-helios platforms

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Since Helios is a rather cumbersome system we should not force ourselves via
governance document to use it.

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge/core
This PR falls under the `Modifying the governance document` policy, please vote and/or comment on this PR.
This PR needs 75% of core to vote yea to pass.
To vote please leave Approve (yea) or Request Changes (nay) reviews.
If you would like changes to the current language please leave a comment or push to this branch.
This vote will end on Friday 15 November 2019.

**Comment by scopatz** [comment]
@conda-forge/core - please vote on this ASAP! (mea culpa, BTW)

File: src/orga/governance.rst [hunk 1]
```diff
@@ -84,8 +84,9 @@ Voting items are labeled as either **standard** or **sensitive**.
 Standard items are ones where public record and discourse is
 preferable. Sensitive voting items are ones where the results of the
 vote should remain private to the voters after the vote has occurred.
-Sensitive votes should take place on
-`the Helios voting system <https://vote.heliosvoting.org/>`_ in order
+Sensitive votes should take place on a secure anonymous voting platform
+(such as `Polys <polys.me>`_ or
+`the Helios voting system <https://vote.heliosvoting.org/>`_) in order
```
Comment (isuruf) [inline src/orga/governance.rst]:
```suggestion
`the Helios voting system <https://vote.heliosvoting.org/>`_) in order to
```

**Comment by scopatz** [comment]
BTW this needs 3 more approvals to pass (14 yea votes)

**Comment by scopatz** [comment]
@conda-forge/core - this needs only one more vote!

**Comment by scopatz** [comment]
This passes!

**Pull request merged**